### PR TITLE
Improve `pick` command to display color styles

### PR DESCRIPTION
**Pull request by hulufei**
eg. [221](https://leetcode.com/problems/maximal-square)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -31,6 +31,8 @@ rand = "0.7.2"
 serde = { version = "1.0.104", features = ["derive"] }
 serde_json = "1.0.44"
 toml = "0.5.5"
+escaper = "0.1.0"
```
Comment (clearloop) [inline Cargo.toml]: You had added an `escaper` at #11 before, bro, why so many `escaper`? LOL~

Comment (hulufei) [reply Cargo.toml]: I'll push a new PR.

Comment (clearloop) [reply Cargo.toml]: It doesn't matter, force push to override is okay.

Comment (hulufei) [reply Cargo.toml]: It's a mess, new PR here #17

**Pull request closed without merging**